src/components/Dropdown/renderOptions.tsx

```diff
@@ -88,7 +89,7 @@ const RenderOptions = <T extends {}>(props: DropdownProps<T>, ref?: React.Ref<HT
             : styles['dropdown-list-positioned-bottom']
         )}
         listItemCategory={listItemCategory}
-        selected={selected}
+        selected={value}
```

Comment:
@higorquintao we're getting a TS error from this line. It seems to be a missing property type warning. Can you please check?